vlogsApp/vlogsApp/Controller/NotificationsScreenViewController.swift

```diff
@@ -6,20 +6,33 @@
 //
 
 import UIKit
+import SnapKit
 
 class NotificationsScreenViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
     
     var tableViewData: [String] = [
     "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
-    "Quisque vel augue eget mi hendrerit faucibus. Pellentesque non eros lacinia, volutpat ligula eu, scelerisque lacus. Nunc ut mi nec nibh egestas aliquam et non sem. Ut dui risus, gravida in lorem a, accumsan mattis ante. Pellentesque placerat ipsum elit. Nulla sodales, mi vel iaculis maximus, nulla quam ullamcorper magna, ut consequat sapien turpis quis justo. Maecenas sed fermentum neque. In ut ligula quis justo tincidunt mattis eget sit amet eros.",
-    "Aliquam et leo mi. Morbi ornare, tellus id auctor imperdiet, mi augue porttitor nisi, ut viverra lorem ipsum vitae dolor. ",
-    "Integer sodales dolor vitae tellus vulputate, sit amet dictum felis lobortis. Sed consequat ante mauris, at iaculis felis placerat ut. Cras mollis quam at lorem molestie porta. Praesent luctus dui vitae est egestas, eu vehicula lorem consectetur. Nam ac consectetur ante, nec convallis lacus. Morbi urna arcu, fermentum ac congue in, lobortis at quam.",
-    "Lorem ipsum"
+    "Quisque et neque metus. Phasellus tortor justo, elementum sed tincidunt in, venenatis sit amet nulla. Mauris libero massa, rhoncus et scelerisque a, interdum at orci. Pellentesque ac sodales turpis, nec porta enim. Sed quis blandit libero. Morbi ornare, est id aliquet eleifend, tortor augue vestibulum orci, ut rutrum dui odio eu mi. Etiam quis lacus enim.",
+    "Aenean viverra rhoncus varius. Sed dolor nulla, dictum vitae elit eu, suscipit aliquet leo. Aliquam id ante et ipsum mollis pulvinar. Nullam dictum mi eu tellus molestie pellentesque. Proin ac neque pellentesque, dignissim arcu tristique, feugiat nibh. Quisque dictum nibh pellentesque, aliquet felis pharetra, dapibus dui. Vestibulum sit amet fermentum tellus. Sed nec odio sed dolor pharetra volutpat.",
```

Comment:
Avoid hardcoded text, If you want to store the notifications and read from databese , make a collection in FirestoreDb and read from there , Make another manager that handles the notifications .